### PR TITLE
feat: rework deploy scripts

### PR DESCRIPTION
**Pull request by skhomuti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,3 +111,28 @@ Kill fork after testing
...
+
+```bash
+just deploy-prod
+```
+
+After that there should be artifacts in the `./configs/latest` directory,
+which is might be moved to the particular directory and commited
+
+```bash
+mv ./configs/latest ./confits/$CHAIN
```
Comment (madlabman) [inline README.md:137]:
```suggestion
mv ./configs/latest ./configs/$CHAIN
```

File: Justfile [hunk 2]
```diff
@@ -103,8 +105,21 @@ kill-fork:
 deploy *args:
     forge script {{deploy_script_path}} --rpc-url {{anvil_rpc_url}} --broadcast --slow {{args}}
 
-deploy-prod:
-    forge script {{deploy_script_path}} --force --rpc-url ${RPC_URL} --broadcast --slow
+deploy-prod *args:
+    just _deploy-prod --broadcast --verify --slow {{args}}
+    mkdir -p ./configs/latest
+    cp ./deploy-latest.json ./configs/latest/deploy-{{chain}}.json
+    cp ./broadcast/{{deploy_script_name}}.s.sol/`cast chain-id`/run-latest.json ./configs/latest/transactions.json
```
Comment (madlabman) [inline Justfile:112]: Need RPC url passed to the cast command.

File: Justfile [hunk 3]
```diff
@@ -103,8 +105,21 @@ kill-fork:
 deploy *args:
     forge script {{deploy_script_path}} --rpc-url {{anvil_rpc_url}} --broadcast --slow {{args}}
 
-deploy-prod:
-    forge script {{deploy_script_path}} --force --rpc-url ${RPC_URL} --broadcast --slow
+deploy-prod *args:
```
Comment (madlabman) [inline Justfile:108]: Need for prompt before running the recipe.

File: Justfile [hunk 4]
```diff
@@ -103,8 +105,28 @@ kill-fork:
...
+    just _warn "The current `tput bold`chain={{chain}}`tput sgr0` with the following rpc url: $RPC_URL"
+    just _deploy-prod-confirm {{args}}
+
+[confirm(
+    "You are about to broadcast the deployment transaction to the network. Are you sure?"
+)]
+_deploy-prod-confirm *args:
+    just _deploy-prod --broadcast --verify --slow {{args}}
+    mkdir -p ./configs/latest
+    cp ./deploy-latest.json ./configs/latest/deploy-{{chain}}.json
```
Comment (vgorkavenko) [inline Justfile:118]: could we deploy to the final path?

Comment (skhomuti) [reply Justfile]: yup fixd

**Pull request merged**